lib/Utils/Feature.php

```diff
@@ -57,11 +96,29 @@ public static function register($feature, $basePath, $options = [])
         }
     }
 
+    /**
+     * Checks if a feature is already registered.
+     *
+     * @since 0.1.0
+     *
+     * @param string $name The feature name.
+     *
+     * @return boolean
+     */
     public static function isRegistered($name)
     {
         return array_key_exists($name, self::$features);
     }
 
+    /**
+     * Gets a feature.
+     *
+     * @since 0.1.0
+     *
+     * @param string $name The feature name.
+     *
+     * @return array
```

Comment:
What does this return exactly? It's neither clear from the description nor explained here in the return keyword.